<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1563</width>
    <height>923</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>仪器设计大赛新生组</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="ball_val_L">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>240</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>160</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="BVL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ball_val_L</string>
    </property>
   </widget>
   <widget class="QLabel" name="arena">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>720</width>
      <height>720</height>
     </rect>
    </property>
    <property name="text">
     <string>arena</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="BX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>BallX:</string>
    </property>
   </widget>
   <widget class="QLabel" name="BY">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>BallY:</string>
    </property>
   </widget>
   <widget class="QLabel" name="TT">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QLabel" name="ballx">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="bally">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ser_selection">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SERL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serial_open">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>131</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>串口状态：</string>
    </property>
   </widget>
   <widget class="QLabel" name="ser_status">
    <property name="geometry">
     <rect>
      <x>503</x>
      <y>131</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serial_close">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cam_open">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>打开相机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ser_update">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>更新串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Xsend</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ysend</string>
    </property>
   </widget>
   <widget class="QLabel" name="X">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Y">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1563</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
